spec/controllers/stores_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe StoresController, type: :controller do
+  describe 'GET /api/v1/tch/stores/:id/directions', type: :request do
+    let(:directions_path) { "/api/v1/tch/stores/#{id}/directions" }
+    let(:address) { '123 Main St' }
+    let(:transportation) { 'driving' }
+    let(:id) { 1 }
+    let(:params) do
+      {
+        address:,
+        transportation:,
+        id:
+      }
+    end
+    let(:success) { true }
+    let(:result) { nil }
+
+    context 'describe #validate' do
+      context 'when address is nil' do
+        let(:address) { nil }
+        before do
+          get(directions_path.to_s, params:)
```

Comment:
why need the to_s ? here, string.to_s ??, please understand what the method does to avoid using redundant or have no impact